docs_sphinx/introduction/install.rst

```diff
@@ -24,9 +24,8 @@ Installation with Anaconda
 Installing Anaconda
 ~~~~~~~~~~~~~~~~~~~
 `Download the Anaconda distribution <https://www.anaconda.com/distribution/#download-section>`_
-for your Operating System. Note that the choice between Python 2.7 and Python
-3.x is not very important at this stage, Anaconda allows you to create a Python
-3 environment from Python 2 Anaconda and vice versa.
+for your Operating System. Note, Brian 2 no longer supports Python 2 (the last
+version to support Python 2 was 2.3). 
```

Comment:
add link

Reply:
to 2.3

Reply:
What link do you mean here?

Reply:
I assumed the relevant section of the release notes so have committed a change pointing to that.

Reply:
Actually, what I was thinking is that once there is only a Python 3 version available we're sure to have people start asking about Python 2 in the support list, and it would be good to have something to point them at. Maybe better than just adding a link to 2.3 would be a subsection of this page explaining how to get the last Python 2 version (and then a link to that here)? That subsection could then have a link to the rtd docs for 2.3 as well as the pip and conda install commands to get Py3 Brian 2.3. What do you think?

Reply:
I think pointing them to the last version supporting it is fine, as we've made it clear that Python 2 is no longer supported. Then if they really need to they can follow the installation instructions to install a specific version i.e. Brian 2.3 but I don't think we should be encouraging anyone to use obsolete software by adding a separate set of instructions, as the time would be better spent reading about `2to3` and porting their old code to Python 3. 

How about merging it in as it is, and consider writing a new subsection if you start getting a lot of questions about it on the support list?

Reply:
OK if you're both happy with it then I'm OK too!

Reply:
Great, going for the merge then!